src/util/tpool/test_tpool.c

```diff
@@ -195,60 +175,28 @@ main( int     argc,
 
   FD_LOG_NOTICE(( "Testing push" ));
 
-  ulong tile_cnt = fd_tile_cnt();
+  ulong tile_cnt = 64;
   tpool = fd_tpool_init( tpool_mem, tile_cnt ); FD_TEST( tpool );
```

Comment:
Kevin, should we just use the number of cpu's on the box?

Reply:
Actually, when I set this to 65, the test gets stuck. I was hoping Kevin could help me figure out why. It was very confusing.